docs/snippets/vue/list-story-unchecked.js.mdx

```diff
@@ -0,0 +1,19 @@
+```js
+// List.stories.js
+
+import List from './List.vue';
+import ListItem from "./ListItem.vue";
+
+// Importing the ListItem stories
+
+import { Unchecked } from './ListItem.stories';
+
+export const OneItem = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { List, ListItem },
+  template: '<List v-bind="$props"><ListItem v-bind="$props"/></List>',
```

Comment:
Do we want to pass the props/args to both the list and the listitem here?

Reply:
This was one of those things that i learned with Vue and Storybook. If i don't add the `v-bind="$props"` it defaults to the default (sorry for the meta here) values that are set in the component and the `args` aren't included.